runtime.json

```diff
@@ -51,23 +51,27 @@
                 "key": "port",
                 "value": "55555"
             },
+            {
+                "key": "port-forward",
```

Comment:
This should be removed if there is no some special reason to have it here, because it is not fully implemented in gen scripts.

Reply:
before removing this we have to check it again with the runtime-local